conformance/base/manifests.yaml

```diff
@@ -25,6 +25,13 @@ metadata:
 ---
 apiVersion: v1
 kind: Namespace
+metadata:
+  name: network-policy-conformance-hogwarts-staff
+  labels:
+    conformance-house: hogwarts-staff
```

Comment:
why do we need a new namespace? IMHO its bloating up the suite, aren't 4 namespaces enough?